common/rabbitmq_cluster/templates/cluster.yaml

```diff
@@ -76,3 +66,34 @@ spec:
               labelSelector:
                 matchLabels:
                   app.kubernetes.io/name: {{ template "fullname" . }}
+            containers:
+            - name : rabbitmq
```

Comment:
whats that entry for?

Reply:
if we didn't add this list item the metrics container will be the default one, which is not the current setup
 i thought that may confuse debugging in case of direct access is need to the containers
unfortunatly we cannot override default-container option as it is not supported in this resource type